icicle/tests/test_curve_api.cpp

```diff
@@ -189,40 +190,21 @@ TEST_F(CurveApiTest, ecntt)
     ntt_release_domain<scalar_t>();
   };
 
-  // run(s_main_target, out_main.get(), "ecntt", VERBOSE /*=measure*/, 1 /*=iters*/);
-  // run(s_ref_target, out_ref.get(), "ecntt", VERBOSE /*=measure*/, 1 /*=iters*/);
-  // ASSERT_EQ(0, memcmp(out_main.get(), out_ref.get(), N * sizeof(projective_t))); // FIXME
+  run(s_main_target, out_main.get(), "ecntt", VERBOSE /*=measure*/, 1);
+  run(s_ref_target, out_ref.get(), "ecntt", VERBOSE /*=measure*/, 1);
+
+  // note that memcmp is tricky here because projetive points can have many representations
+  for (uint64_t i = 0; i < N; ++i) {
+    ASSERT_FALSE(projective_t::is_zero(out_ref[i]));
+    ASSERT_EQ(out_ref[i], out_main[i]);
+  }
 }
 
-TEST_F(CurveApiTest, ecntt_new)
+TEST_F(CurveApiTest, ecnttRandTest)
 {
-  // Randomize configuration
-
-  // int seed = time(0);
-  // srand(seed);
-  // const bool inplace = rand() % 2;
-  // const int logn = rand() % 15 + 3;
-  // const uint64_t N = 1 << logn;
-  // const int log_ntt_domain_size = logn + 1;
-  // const int log_batch_size = rand() % 3;
-  // const int batch_size = 1 << log_batch_size;
-  // const Ordering ordering = static_cast<Ordering>(rand() % 4);
-  // bool columns_batch;
-  // if (logn == 7 || logn < 4) {
-  //   columns_batch = false; // currently not supported (icicle_v3/backend/cuda/src/ntt/ntt.cuh line 578)
-  // } else {
-  //   columns_batch = rand() % 2;
-  // }
-  // const NTTDir dir = static_cast<NTTDir>(rand() % 2); // 0: forward, 1: inverse
-  // const int log_coset_stride = rand() % 3;
-  // scalar_t coset_gen;
-  // if (log_coset_stride) {
-  //   coset_gen = scalar_t::omega(logn + log_coset_stride);
-  // } else {
-  //   coset_gen = scalar_t::one();
-  // }
+  // (TODO) Randomize configuration
```

Comment:
You missed a TODO

Reply:
It's for Shanie, I just removed the commented out code she left